LEAF_Request_Portal/admin/templates/mod_workflow.tpl

```diff
@@ -2177,85 +2181,89 @@
      * Created at: 7/26/2023, 1:08:10 PM (America/New_York)
      */
     function duplicateWorkflow() {
-        $('.workflowStepInfo').css('display', 'none');
+        if (+currentWorkflow > 0) {
+            $('.workflowStepInfo').css('display', 'none');
 
-        dialog.setTitle('Duplicate current workflow');
-        dialog.setContent('<br /><label for="description">New Workflow Title:</label> <input type="text" id="description"/><br /><br />The following will NOT be copied over:<br /><br />&nbsp;&nbsp;&nbsp;&nbsp;Data fields that show up next to the workflow action buttons');
-        dialog.setSaveHandler(function() {
-            let old_steps = {};
-            let workflowID;
-            let title = $('#description').val();
+            dialog.setTitle('Duplicate current workflow');
+            dialog.setContent('<br /><label for="description">New Workflow Title:</label> <input type="text" id="description"/><br /><br />The following will NOT be copied over:<br /><br />&nbsp;&nbsp;&nbsp;&nbsp;Data fields that show up next to the workflow action buttons');
```

Comment:
This doesn't make a lot of sense.  This is a very specific error message and is static.  When you say "the following", you expect to be more than two.   Thinking this should be run by Michael Gao and Su as I'm not sure of better wording myself.

Reply:
This was deployed half a year ago